zerver/tests/tests.py

```diff
@@ -201,19 +219,23 @@ def test_admin_api(self):
 class WorkerTest(TestCase):
     class FakeClient(object):
         def __init__(self):
-            self.consumers = {}
-            self.queue = []
+            # type: () -> None
+            self.consumers = {} # type: Dict[str, Callable]
+            self.queue = [] # type: List[Dict[str, Any]]
```

Comment:
Good catch, I hadn't noticed that this mock queue client had a different style of queue than all of our actual queues.

It's a bit surprising that mypy didn't catch this.